server/controllers/personal/personalController.ts

```diff
@@ -150,10 +166,35 @@ export default class PersonalController {
           const feet = parseInt(feetString, 10)
           const inches = parseInt(inchesString, 10)
 
-          if (Number.isNaN(feet) || feet <= 0 || Number.isNaN(inches) || inches <= 0) {
+          const validations = [
+            {
+              isInvalid: () =>
+                ((feetString === '' || feetString === undefined) &&
+                  (inchesString === '' || inchesString === undefined)) ||
```

Comment:
of these are strings can we not just do `!feetString || !inchesString` ?

Reply:
Also, I think that if it was undefined then parseInt would have already errored and if it was '' then it would have been NaN so perhaps this isn't doing much

Reply:
I had incorrectly presumed '0' gets coerced into false but i've just double checked and that's not the case so I will update this